worlds/generic/docs/advanced_settings_en.md

```diff
@@ -158,7 +157,7 @@ game:
   Timespinner: 10
 requires: 
   version: 0.3.2
-accessibility: none
+accessibility: minimal
 progression_balancing: on
```

Comment:
`accessibility`, `progression_balancing`, (and `triggers` probably) should be moved into the game key instead of a root-level option.

Also `progression_balancing` should be a number now between 0 and 99. 50 is default.